code-challenges.js

```diff
@@ -64,12 +194,54 @@ var refactorTester3 = "IAmACodingMaster"
 var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
 var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
 
-
-
+//create a function called longestString
+//takes in an array of strings
+const longestString = (array) => {
+    //create a variable that stores the lengths of each string in each array
+    let max = array.length
+    // combining the map and math.max functions to iterate though each array and find the length of each
+    array.map(value => max = Math.max(max, value.length))
+    //using filter to return a new array with the longest string out of the previous array
+    return array.filter(value => value.length === max)
```

Comment:
great solution because it can allow the function to return multiple words if they are of the same (max) length